plugins/parquet/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 ##############################################################################
 
-#    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+#    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems�.
```

Comment:
Is this change correct?

Reply:
I didn't make that change myself, and I assumed that some editor was fixing it.  On my system, it now looks like:

#    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.

That is Unicode character 00AE.

I just compared it with the same character in plugins/kafka/CMakeLists.txt and it is the same (00AE).

I think it's correct, just not rendered correctly here.

Reply:
Ok, that makes sense.

Reply:
I just realized that, in my previous comment, I was comparing characters in a difference git branch.  After switching to the right branch I see that this character is wrong.

I will revert this change in this file.